pages/movies/index.js

```diff
@@ -11,12 +22,12 @@ export async function getStaticProps() {
         props: {
             posters: posters.map((poster) => ({
                 title: poster.title,
-                id: poster._id.toString()
+                poster: poster.poster,
+                id: poster._id.toString(),
             })),
         },
-        revalidate: 1
+        revalidate: 1,
```

Comment:
To revalidate every second makes little sense. It means content is only cached for one second, and you wouldn't really notice a difference unless you have significantly more than one user per second. For less than one user per second you might actually see performance _degrade_ as generating the static page (and writing it to cache) might take slightly longer than generating a dynamic one on-demand.